concordium-consensus/src/Concordium/GlobalState/Parameters.hs

```diff
@@ -101,7 +101,9 @@ data RuntimeParameters = RuntimeParameters {
   -- transaction table if a purge is executed.
   rpTransactionsKeepAliveTime :: !TransactionTime,
   -- |Number of seconds between automatic transaction table purging  runs.
-  rpTransactionsPurgingDelay :: !Int
+  rpTransactionsPurgingDelay :: !Int,
+  -- |The maximum allowed time difference between slot time and a transaction's expiry time.
+  rpMaxTimeToExpiry :: !TransactionTime
```

Comment:
This would ideally be `Duration`, but `rpTransactionKeepAliveTime` should also be that, so I kept it consistent with that so that I don't introduce accidental bugs.